Assets/AppCenter/Plugins/AppCenterSDK/Core/Shared/StartupType.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+namespace Microsoft.AppCenter.Unity
+{
+    /// <summary>
+    /// SDK Startup mode.
+    /// </summary>
+    public enum StartupType
+    {
+        /// <summary>
+        /// The app starts with AppCenter.start() SDK call using an appSecret that is targeting AppCenter. 
+        /// If you send events in the app it will thus go to App Center portal. 
+        /// </summary>
+        AppCenter = 0,
+
+        /// <summary>
+        /// Only Analytics is usable, other modules appear disabled. 
+        /// Which maps to calling AppCenter.start(“target={oneCollectorToken}“, modules).
+        /// </summary>
+        OneCollector = 1,
+
+        /// <summary>
+        /// Events go to OneCollector, crashes go to AppCenter. 
+        /// It maps to AppCenter.start(“target={oneCollectorToken};appsecret={appSecret}“, modules).
```

Comment:
Please remove the following symbol from the end of the line: 

Reply:
`“` -> `"`